tests/test_models/test_review.py

```diff
@@ -118,6 +118,6 @@ def test_to_dict_values(self):
 
     def test_str(self):
         """test that the str method has the correct output"""
-        review = Review()
-        string = "[Review] ({}) {}".format(review.id, review.__dict__)
-        self.assertEqual(string, str(review))
+        r = Review()
+        string = "[Review] ({}) {}".format(r.id, r.__dict__)
+        self.assertEqual(string, str(r))
```

Comment:
Many of the same issues as City. Rinse and repeat.

Reply:
Roger that!